### ISSUE TITLE
Suggested improvements to `9-variance.pdf` 

### ISSUE THREAD

**Issue by jjrob** [DistanceDevelopment/spatial-workshops#38]
Just some minor considerations:

1. Slides 18 and 19 should be reversed: you should show the `dsm.var.gam` result before `dsm.var.prop`, so you can show how variance increased with the `dsm.var.prop` approach. Also, it is better "parallel construction" to have `dsm.var.gam` come first, since it also did on slide 17 immediately prior.

2. It is often the case with these models that CV will be very high in areas where predicted abundance is very low. Slides 22-24 should discuss this issue and possibly the pros and cons of CV vs. SE. While this may arguably be a "basic stats" thing that people should already understand, many probably won't and it would be very helpful to them if we briefly exposed them to it.

3. In the "habitat modeling" branch of species distribution modeling methodologies, there is a long tradition of having a "training set" and a "test set". See, for example, the classic Guisan and Zimmerman (2000). Distance sampling does not have this tradition, and instead has a tradition of variance estimation and also cross validation that might be done behind the scenes by mgcv. Attendees who come from the "habitat modeling" background will want to know why you don't have a training and test set, and may consider your model kind of unproven without one. This is worth some high level philosophical discussion.

**Comment by dill** [comment]
Thanks for these comments:

1. Fixed in db8368e. Note sure what you mean about "Also, ..." here...
2. I've address plotting large CVs in 4595ee9. Maybe we can also include this in "pratical advice"?
3. This is in the practical advice lecture already, but I'm happy to expand that section.
